src/components/QnA/comps/ModalTemplate.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import styled from 'styled-components';
+import FlexColumn from '../styles/FlexColumn.styled';
+
+const OuterFlexColumn = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: auto;
+`;
+
+const FlexForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100%;
+  max-width: none;
+`;
+
+const ModalTitle = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const ModalSubtitle = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const Label = styled.label`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+`;
+
+const FirstInput = styled.textarea.attrs({ maxLength: '1000', required: 'required' })`
+display: block;
+padding: 0 0 15px 0;
+width: 100%;
+overflow: auto;
+height: 65px;
+line-height: 1.2;
+`;
+
+const SecondInput = styled.input.attrs({ maxLength: '60', required: 'required' })`
+padding: 0 0 15px 0;
+`;
+
+const ThirdInput = styled.input.attrs({ maxLength: '60', required: 'required', type: 'email' })`
```

Comment:
We can set attributes within the JSX, so there isn't a need to make a `ThirdInput` component. We can probably just use `<SecondInput type="email" />` instead.